scripts/data/observations_to_datastore.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python
+
+from astropy import units as u
+from astropy.time import Time
+
+from google.cloud import datastore
+
+from pocs.base import PanBase
+from pocs.utils import current_time
+
+
+class ObservatonsExporter(PanBase):
+    """ A simple class for exporting observations to Google Datastore
+
+    Observations are pulled from the corresponding mongo collection.
```

Comment:
Perhaps add a bit more theory of operation here, or more comments throughout? I can't immediately see how it works (well, I get the basic idea, but not enough).

Reply:
This was mostly moving my experimenting into something more permanent. I am going to start a team Discussion to see if the format is what we want. Also, I want to make a PR today that will put the records into Datastore in real-time, which should be much more obvious. I hesitated about even pushing this up.